packages/dds/tree/src/test/shared-tree/fuzz/composeVsIndividual.fuzz.spec.ts

```diff
@@ -148,24 +170,42 @@ describe("Fuzz - composed vs individual changes", () => {
 			DDSFuzzTestState<SharedTreeTestFactory>
 		> = {
 			workloadName: "SharedTree",
-			factory: new SharedTreeTestFactory(() => {}),
+			factory: new SharedTreeTestFactory(createOnCreate(undefined)),
 			generatorFactory,
 			reducer: fuzzComposedVsIndividualReducer,
 			validateConsistency: () => {},
 		};
 		const emitter = new TypedEventEmitter<DDSFuzzHarnessEvents>();
 		emitter.on("testStart", (initialState: BranchedTreeFuzzTestState) => {
 			initialState.main = viewFromState(initialState, initialState.clients[0]);
-			initialState.branch = initialState.main.fork() as FuzzTransactionView;
-			initialState.branch.currentSchema = initialState.main.currentSchema;
+
+			const branchCheckout = initialState.main.checkout.fork();
+			const treeSchema = simpleSchemaFromStoredSchema(
+				initialState.main.checkout.storedSchema as TreeStoredSchemaRepository,
+			);
+			const branchView = viewCheckout(
+				branchCheckout,
+				new TreeViewConfiguration({ schema: treeSchema }),
+			) as FuzzTransactionView;
+
+			const nodeSchema = nodeSchemaFromTreeSchema(treeSchema);
+
+			branchView.currentSchema =
+				nodeSchema ?? assert.fail("nodeSchema should not be undefined");
+			initialState.branch = branchView;
```

Comment:
could all this setup share code with above?

Reply:
Removed logic from code above, as we decided to remove schemaOps from the composeVsIndividual tests